apps/oxlint/src-js/plugins/source_code.ts

```diff
@@ -118,7 +124,9 @@ export const SOURCE_CODE = Object.freeze({
 
   // Get visitor keys to traverse this AST.
   get visitorKeys(): { [key: string]: string[] } {
-    throw new Error('`sourceCode.visitorKeys` not implemented yet'); // TODO
+    // This is the path relative to `plugins.js` file in `dist` directory
+    if (visitorKeys === null) visitorKeys = require('./generated/visit/keys.js').default;
```

Comment:
nit: could probably use the newer es syntax

```suggestion
    visitorKeys ??= require('./generated/visit/keys.js').default;
```